examples/browser-api-playground/src/components/Layout/PageLayout.tsx

```diff
@@ -1,18 +1,36 @@
-import { PropsWithChildren } from "react";
+import { PropsWithChildren, useEffect, useState } from "react";
 import Header from "./Header";
-import { EmbedConfigContext } from "../../EmbedConfigContext";
+import { EmbedConfigContext, sdkOptionsKey } from "../../EmbedConfigContext";
 import useConfigStore from "../../useConfigStore";
 import { Layout } from "antd";
 import { Content } from "antd/es/layout/layout";
 
 const PageLayout = ({ children }: PropsWithChildren) => {
+  const [ready, setReady] = useState(false)
   const contextValue = useConfigStore()
 
+  useEffect(() => {
```

Comment:
Would it be cleaner to have this embedded-search script injection in `index.tsx`? Also, we can return null if ready is false.

Reply:
I had that initially, but moved here since its only blocking the `Content` here not the other parts like `Header` etc

Reply:
Makes sense

Reply:
ya the same reason, since return null here will just render nothing, while its still legit to render Header ;)

Reply:
Makes sense.